apps/example/src/App.jsx

```diff
@@ -4,6 +4,7 @@
  */
 
 import React, { useEffect } from "react"
+import PropTypes from "prop-types"
 import styles from "./styles.scss"
 
 console.log("====================", React)
```

Comment:
I think would be nice to have a rule that forbid console.log in a way we use it, let us also maybe investigate on it. let me know what do you think?
https://eslint.org/docs/latest/rules/no-console